### DISCUSSION TITLE
Radially oriented elongated grains

### DISCUSSION THREAD

**Discussion by avtarsinghh1991** [neperfepx/neper#430] [Q&A] [answered]
Hello NEPER Experts

I am new to neper. I am wondering if I can able to generate the grains with specific orientation given to individual grains along with the given average aspect ratio as shown in figure. Please help.

I am referring to the anisotropic Voronoi tessellation in which aspect ration and orientation of individual Voronoi cell can be defined.

I found the following command. Is it useful to do that?

`oriaverage: set the cell orientations (field **cell/ori) as the averages of the cell voxel orientations (field **oridata).`
 
Thanks in advance.

Best
Avtar
![Untitled](https://user-images.githubusercontent.com/94848840/162052121-40423161-ccda-478c-90d9-444529d1e0e3.png)

**Comment by rquey** [comment, marked as answer]
If you mean radially oriented grains, it can be obtained using a circular seed distribution. You'd have to define it yourself (outside Neper), write to a file, and load the seed distribution using `-morphooptiini coo:file(...)`.

If you mean grains with high elongation ratios and random distributions, Neper does not really allows it yet.
